generators/sculpt.js

```diff
@@ -205,7 +205,7 @@ export function sculptToGLSL(userProvidedSrc) {
 	userProvidedSrc = replaceMathOps(userProvidedSrc);
 
 	if (debug) {
-		console.log('tree', tree);
+		console.log('userProvidedSrc', userProvidedSrc);
```

Comment:
Originally this would have printed the parsed source tree that is generated by esprima. I'm not sure if printing the user source here is that helpful. Tbh the debug flag has not really been maintained since it was created and we might want to completely visit it or just remove it